### DISCUSSION TITLE
TS0201 Temperature Unit Change

### DISCUSSION THREAD

**Discussion by errolsancaktar** [Koenkk/zigbee2mqtt#19043] [Devices]
I've search everywhere i can think of. i found via homeseer that there is apparently a way to change the temperature unit display on the ts0201/0601 device.  I dont fully understand how the configuration of this works but it looks like there is a clusterid/atribute that can be modified.  is there any way to do that via mqtt?  here is what i found:

```
    '_TZ3000_itnrsufe'  : 'TS0201_TH',          // Temperature and humidity sensor; // reports both battery voltage and perceintage; cluster 0xE002, attr 0xE00B: 0-Celsius, 1: Fahrenheit ( 0x30 ENUM)
```

https://raw.githubusercontent.com/kkossev/Hubitat/development/Drivers/Tuya%20Temperature%20Humidity%20Illuminance%20LCD%20Display%20with%20a%20Clock/Tuya_Temperature_Humidity_Illuminance_LCD_Display_with_a_Clock.groovy

**Comment by stashaway** [comment]
I'm looking for the same answer. It's configurable in the Tuya app, but I can't figure it out in HA/Z2MQTT
